packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -1587,6 +1587,117 @@ function CustomDragPreviewExample() {
 <CustomDragPreviewExample />
 ```
 
+## Expandable Rows <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+By default, TableView doesn't allow expandable rows, but this can be enabled using the `hasExpandableRows` prop.
+You'll first need to enable a feature flag in your app to use this feature.
+
+```tsx example
+import {enableTableNestedRows} from '@react-stately/flags';
+
+enableTableNestedRows();
+```
+<br />
+
+Use `defaultExpandedKeys` to provide a default set of expanded rows. Note that the value of the expanded keys must match the `key` prop of the Row and that a
+expanded child row will only be visible if its parent row key is also included within the expanded key set.
+
+A expandable row can be statically defined by including one or more Rows as children. Note that all child Rows must be placed after the last Cell within the Row.
```

Comment:
I noticed that sometimes the word "row" is capitalized and sometimes it isn't (particularly in the paragraph above). It would probably be good to keep it consistent one way or another. So like either "child Row" or "child row" rather than switching between the two.

Reply:
So the difference between the two was to try to distinguish between referencing the component Row/Cell and the row/cell generically. I'll add back ticks around the capital Rows to make it more clear